sixtyfps_compiler/builtins.60

```diff
@@ -568,9 +568,9 @@ export NativeTab := _ {
 }
 
 export global NativeStyleMetrics := {
-    property <length> layout_spacing;
-    property <length> layout_padding;
-    property <length> text_cursor_width;
+    property <length> layout-spacing;
+    property <length> layout-padding;
+    property <length> text-cursor-width;
     //-is_non_item_type
     //-is_internal
 }
```

Comment:
Just a thought, not relevant for this change: When loading `builtins.60` we could try to debug_assert that all property names/callbacks are free of underscores.

Reply:
we could...  but that's just style, this file is normalized anyway.

Another thought is that we could assert that in the documentation or in the example.